app/queries/submission_details_resolver.rb

```diff
@@ -19,7 +20,8 @@ def submission_details
       course_id: level.course_id,
       created_at: submission.created_at,
       preview: preview?,
-      reviewer_details: reviewer_details
+      reviewer_details: reviewer_details,
+      submission_report_poll_time: ENV.fetch('SUBMISSION_REPORT_POLL_TIME', 30)
```

Comment:
Can we read the `env` from `Rails.application.secrets`, the default value for the variable could be configured in the `secrets.yml` fiel
```suggestion
      submission_report_poll_time: Rails.application.secrets.submission_report_poll_time
```